install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,19 +40,27 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: ClusterVersionOperatorCannotRetrieveUpdates
+      annotations:
+        message: Cluster version operator has not retrieved updates for 10 minutes. Failure reason {{ "{{ $labels.reason }}" }}. {{ "{{ $labels.message }}" }}.
+      expr: |
+        cluster_version_operator_retrieving{job="cluster-version-operator"} == 0
+      for: 10m
```

Comment:
10 minutes seems overly strict given our loose Cincinnati service-level agreements and objectives.  Can we make this at least an hour?